__tests__/unittest/date.test.js

```diff
@@ -0,0 +1,24 @@
+const { getRFC3339Date } = require('../../lib/date');
+
+const RFC3339_REGEX = /^((?:(\d{4}-\d{2}-\d{2})T(\d{2}:\d{2}:\d{2}(?:\.\d+)?))(Z|[+-]\d{2}:\d{2})?)$/;
```

Comment:
```suggestion
const RFC3339_REGEX = /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(?:\.\d+)?(?:Z|[+-]\d{2}:\d{2})$/;
```

Reply:
Hi @aramovic79 , good catch. I try to use validator to check the lastUpdate without offset `2024-06-20T14:04:01` and it passed. So I'm curious, how do you know lastUpdate can be without offset or do you know somewhere use it as well? Thx

Reply:
According to the RFC 3339 datetime-format definition, if there is no offset(so, the time-offset is 0), the letter **Z** should stand at the end, to show that the datetime is UTC. 
Regex should not "allow" a datetime without time-offset or letter Z at the end. 
If my proposed regex allows it, forget it. 😊

Reply:
This should match the date format in the ORD spec, but there we just refer to https://datatracker.ietf.org/doc/html/rfc3339#section-5.6 and validate it as a JSON Schema "format": "date". I'm not entirely sure what the regexp of JSON Schema dates looks like, but we should compare this to make sure we got the same validation and understanding.

Reply:
Hi @Fannon , the unittest cases are taken from [rfc3339#section-5.8 examples](https://datatracker.ietf.org/doc/html/rfc3339#section-5.8). I think the current implementation should be fine, later we can use validator to check the correctness of ord json format.